Breaking Release Documentation Improvements and Fixes (#2191)
ethereumjs/ethereumjs-monorepo: README.md

[![Discord][discord-badge]][discord-link]
[![GitPOAP][gitpoap-badge]][gitpoap-link]

This repository holds various protocol building blocks of the [Ethereum](https://ethereum.org) blockchain execution layer and is managed by the [Ethereum Foundation](https://ethereum.foundation/) JavaScript team. There is a TypeScript implementation of the [Ethereum Virtual Machine (EVM)][vm-package] ready to be used in Node.js or a browser environment, implementations of core structural blockchain building blocks like an Ethereum [Tx][tx-package], [Block][block-package] or [Blockchain][blockchain-package] as well as a [Trie][trie-package] (Merkle Patricia Tree) and [devp2p][devp2p-package] (execution networking layer) implementation.

> All libraries are bound together by the core [Common][common-package] library keeping track of chain specifics and hardfork changes and are complemented by helper packages like [RLP][rlp-package] for data encoding/decoding or [Util][util-package] with helper functionality for (byte) conversion, signatures, types and others.

Reviewer: ```suggestion
All libraries are bound together by the core [Common][common-package] library keeping track of chain specifics and hardfork changes. They are complemented by helper packages like [RLP][rlp-package] for data encoding/decoding or [Util][util-package], providing helper functionalities like (byte) conversion, signatures, types and others.
```
I think breaking the sentence up in two makes it a bit clearer.

> Finally the [EthereumJS Execution Client][client-package] (EthereumJS) is in active development for some time now and already servers a variety of purposes regarding testing, in the area of (EIP) research as well as a developer tool to engage with the protocol.

Reviewer: ```suggestion
Finally, the [EthereumJS Execution Client][client-package] (EthereumJS) has been in active development for some time now. It already serves a variety of purposes like testing, research (e.g. EIPs) and developer tooling to engage with the protocol.
```

> Also to note: a repository very much complementing this repository on the Ethereum Consensus side is the ChainSafe [Lodestar](https://github.com/ChainSafe/lodestar) repository with an Ethereum Consensus Client implementation as well as various protocol implementations (like an SSZ library) developed in the context of Ethereum Consensus layer evolution.

Reviewer: ```suggestion
Also to note: on the Ethereum Consensus side, the ChainSafe [Lodestar](https://github.com/ChainSafe/lodestar) repository complements this repository with an Ethereum Consensus Client implementation as well as various protocol implementations (like an SSZ library) developed in the context of Ethereum Consensus layer evolution.
```

| [master](https://github.com/ethereumjs/ethereumjs-monorepo)                             | v6             | `Beta`   | Main working branch, current v6 work merged in (2022-06-08) |
| [v5-maintenance](https://github.com/ethereumjs/ethereumjs-monorepo/tree/v5-maintenance) | v5             | `Active` | Maintenance for v5 releases                                 |

> Breaking releases are in sync for all libraries, a release cycle is named after the `@ethereumjs/vm` version. In most cases PRs should be opened towards the current working branch.

Reviewer: ```suggestion
Breaking releases are in sync for all libraries, and release cycles are named after the `@ethereumjs/vm` version. In most cases PRs should be opened towards the current working branch.
```
